components/org.wso2.carbon.identity.conditional.auth.functions.choreo/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/choreo/CallChoreoFunctionImpl.java

```diff
@@ -57,6 +57,8 @@
 
     private static final String URL_VARIABLE_NAME = "url";
     private static final String API_KEY_VARIABLE_NAME = "apiKey";
+    private static final String API_KEY_ALIAS_VARIABLE_NAME = "apiKeyAlias";
+    private static final String SECRET_TYPE = "adaptive";
```

Comment:
Can we rethink this SECRET_TYPE name? `adaptive` seems to be a bit generic 
What about `CALL_CHOREO_ADAPTIVE_SCRIPT`

Reply:
Resolved